source/Patches/Roles/Role.cs

```diff
@@ -488,17 +488,31 @@ public static void Postfix(ref string __result, [HarmonyArgument(0)] StringNames
             {
                 if (ExileController.Instance == null || ExileController.Instance.exiled == null) return;
 
+                // If Lover was Voted and option was activated we display the role and the other lover in the ejection text to prevent other players to replace the body appaerant
```

Comment:
Can this live somewhere in LoversMod folder? Dumping everything into Role.cs forces us to have really bad variable scope (everything has to be public) and it's hard to figure out what code controls what.